### PR TITLE
Thanks for your great work improving this theme RBertoCases! I made some minor tweaks for myself and thought I'd share

### PR DESCRIPTION
**Pull request by timbiscuits**

### CODE REVIEW

File: layer_grid/SoftwareScreen.qml [hunk 1]
```diff
@@ -210,16 +210,17 @@ FocusScope
 
             anchors {
                 left: parent.left; leftMargin: vpx(63)
-                top: topBar.bottom;
+                top: topBar.bottom
                 right: parent.right; rightMargin: vpx(63)
                 bottom: parent.bottom
             }
+			topMargin: Math.round(screenheight*0.1)
```
Comment (RBertoCases) [inline layer_grid/SoftwareScreen.qml:217]: Love it!

File: layer_grid/SoftwareScreen.qml [hunk 2]
```diff
@@ -247,7 +248,7 @@ FocusScope
                         
                         asynchronous: true
                         //smooth: true
-                        source: modelData.assets.screenshots[0] ? modelData.assets.screenshots[0] : ""
+                        source: modelData.assets.background
```
Comment (RBertoCases) [inline layer_grid/SoftwareScreen.qml:251]: I get what you're going for, but as someone who doesn't have fanart set for all my games, but has screenshots, I prefer screenshots. Maybe I can do an option in the future to toggle which to display, backgrounds instead screenshots

Comment (timbiscuits) [reply layer_grid/SoftwareScreen.qml:251]: No stress! I've come from gameOS which pulls background & logo for everything 👍

File: layer_platform/PlatformBar.qml [hunk 3]
```diff
@@ -134,7 +134,7 @@ ListView {
                 height: width
                 smooth: true
                 fillMode: Image.PreserveAspectCrop
-                source: gameData ? gameData.assets.screenshots[0] || "" : ""
+                source: gameData.assets.background
```
Comment (RBertoCases) [inline layer_platform/PlatformBar.qml:137]: Same deal as the comment on the SoftwareScreen - I get what you're going for, but as someone who doesn't have fanart set for all my games, but has screenshots, I prefer screenshots. Maybe I can do an option in the future to toggle which to display, backgrounds instead screenshots

File: layer_grid/SoftwareScreen.qml (hunk 2 above)
Comment (RBertoCases) [reply layer_grid/SoftwareScreen.qml:251]: Right, I'll probably look at gameOS when I go to add that functionality. But it has a preference for the backgrounds depending on the context that might be improved if it's a user toggle-able. Plus, need an excuse to get the buttons between the platformbar and controller help section working. Maybe one of those buttons will be for a Settings screen

**Review by RBertoCases — COMMENTED** [review]
Oh man, thank you for sharing! I couldn't comment on the profile_icons, but I'm going to keep those for now because I'm not sure when I'd get to that

**Pull request closed without merging**